packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Asset/Asset.tsx

```diff
@@ -480,7 +485,12 @@ const NftAsset: React.FC<Props> = ({
                                 />
                               ) : (
                                 <Flex alignItems='center' flexDirection='row'>
-                                  <SkeletonCircle height='34px' width='34px' bgColor='grey000' />
+                                  <Avatar
+                                    size={34}
+                                    name={owner.address || ''}
+                                    variant='marble'
+                                    colors={['#0C6CF2', '#5322E5', '#F00699', '#06D6A0', '#121D33']}
```

Comment:
why hardcoded?

Reply:
just some of the colors weren't in constellation.  changing to closest ones